core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.ts

```diff
@@ -85,29 +90,25 @@ angular.module('oppia').controller('QuestionSuggestionReviewModalController', [
     $scope.reviewMessage = '';
     $scope.suggestionIsRejected = suggestion.status === 'rejected';
 
-    const _getThreadHandlerUrl = function(suggestionId) {
-      return UrlInterpolationService.interpolateUrl(
-        '/threadhandler/<suggestionId>', { suggestionId });
+    var _getThreadMessagesAsync = function(threadId) {
+      return ThreadDataBackendApiService.fetchMessagesAsync(
+        threadId).then((response) => {
+        let threadMessageBackendDicts = response.messages;
```

Comment:
Doesn't look like this variable gets reassigned, so would prefer `const`.

Reply:
Done